File: src/baekjoon/step/bruteForce/PROB1436.java
package baekjoon.step.bruteForce;

import java.io.*;

public class PROB1436 {

    /**
     * 666 이 들어간 숫자들을 순서대로 나열하면...
     * {666, 1666, 2666 ... 5666, 6660, 6661 ... 6669, 7666, 8666, ... }
     */

    static BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
    public static void main(String[] args) throws IOException {
        System.out.println(findNth666Series(Integer.parseInt(br.readLine())));
    }
    public static int findNth666Series (int n) {
        int loop = n;
        int number = 665;
        while (loop > 0) {
            number++;
            if (String.valueOf(number).contains("666")) {
                loop--;
            }
        }
        return number;
    }
}

File: src/baekjoon/step/bruteForce/PROB1436.py
# 제일 작은 종말의 수 : 666
# 종말의 수 : 어떤 수에 6이 적어도 3개 이상 들어가는 수
# N 은 10000 보다 작거나 같은 수이다.


# N 입력받기
N = int(input())

# N 번째로 작은 종말의 숫자 구하기
number = 666
count = 0

while True :
    if "666" in str(number) :
        count += 1

    if count == N :
        print(number)
        break

    number = number + 1